packages/snaps-cli/src/cmds/build/bundle.ts

```diff
@@ -25,6 +25,7 @@ export function bundle(
   src: string,
   dest: string,
   argv: YargsArgs,
+  bundlerTransform?: (bundler: BrowserifyObject) => void,
```

Comment:
Perhaps this should just be `SnapConfig` so we can use other options in the future?

Reply:
The build system shouldn't know about how we configure the cli system, that would couple them both. In the future if we want to add more options we can change into `options` object